packages/pages/src/vite-plugin/modules/plugin.ts

```diff
@@ -171,3 +205,60 @@ const removeAddedModuleCode = (modulePath: string, index: number) => {
   sfp.removeUnusedImports();
   sfp.save();
 };
+
+/**
+ * Returns the postcss.config filepath if it exists in module.
+ * Else returns the root postcss.config filepath.
+ * If there is none, throws error.
+ *
+ * @param rootFolders
+ * @param subfolders
+ * @param filename of module
+ * @returns string
+ */
+const getPostcssConfigFilepath = (
+  rootFolders: any,
+  subfolders: any,
+  filename: string
+): string => {
+  for (const extension of postcssExtensions) {
+    const filePath = path.join(
+      rootFolders.source,
+      subfolders.modules,
+      `${filename}/postcss.config.${extension}`
+    );
+    if (fs.existsSync(filePath)) {
+      return filePath;
+    }
+  }
+
+  for (const extension of postcssExtensions) {
+    const filePath = path.join(`postcss.config.${extension}`);
+    if (fs.existsSync(filePath)) {
+      return filePath;
+    }
+  }
+
+  throw new Error(`Cannot find a postcss.config file for ${filename}`);
+};
+
+/**
+ * Looks at all css files in src and returns true
+ * if there is an unwrapped tailwind base
+ *
+ * @returns boolean
+ */
+const tailwindBaseExists = (): boolean => {
+  const files = glob.sync("./src/**/*.css");
+  for (const filePath of files) {
+    try {
+      const data = fs.readFileSync(filePath, "utf8");
+      if (data.includes(`@tailwind base`) && !data.includes(`.tailwind {`)) {
+        return true;
+      }
+    } catch (err) {
+      throw new Error(`Cannot read file at ${filePath} due to: ${err}`);
```

Comment:
I think we ignore the error and return false. A user could have some weird file we effectively just want to skip over. Comment about this in the catch though because it's intentional we're not handling.